docs/docs/tutorials/basic_arithmetic_grpc_server.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+title: Creating a Basic Arithmetic gRPC Server
+parent: Tutorials
+nav_order: 3
+---
+
+## Creating a Basic Arithmetic gRPC Server
+
+First look at the `main.go` file of Arithmetic server that will serve on a given port and listening at the endpoint we defined in proto files.
```

Comment:
Is this right? I don't think the `Server` is defined in `main.go` in your arithmetics server, and generally this sort of struct would be defined in a separate package from `main`. I assume you mean to talk about the `server/server.go` file here.